tests/unit/apps/test_app.py

```diff
@@ -16,8 +16,8 @@
 
 import pytest
 
-from cou.apps import app as app_module
-from cou.apps.app import OpenStackApplication
+from cou.apps import factory
```

Comment:
Please, create a new file `test_factory.py` and transfer all factory tests to this new file